tests/ui/check-cfg/unexpected-cfg-name.names.stderr

```diff
@@ -0,0 +1,10 @@
+warning: unexpected `cfg` condition name
```

Comment:
```suggestion
warning: unexpected `cfg` condition name: widnows
```
This is also an independent change, but could you mention the unexpected names and values in the primary messages as well?
(Could be split to a separate PR together with `should_print_possibilities`.)

Reply:
Done as part of https://github.com/rust-lang/rust/pull/116666.